.github/workflows/.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": [
+          "tailwind",
+          "apply",
+          "variants",
+          "responsive",
+          "screen"
+        ]
+      }
+    ]
+  },
+  "csstree/validator": true,
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+}
+

```

Comment:
- [ ] [OPTIONAL] Please make sure to remove this file, this folder `.github/workflows` is supposed to only host your `linters.yml` file which is the _only_ config file that you need in order to set up linters on GitHub, `.stylelintrc.json` is a local config file that is used to set up linters on your local machine and should be in the root directory if you wish to add to your repository.